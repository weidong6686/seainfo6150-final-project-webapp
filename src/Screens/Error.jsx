import React from 'react';
import { Link } from "react-router-dom";
import styles from './css/Error.css';

const Error = () => {
    return (
        <div className={styles.container}>
            <h1>ERROR 404 - Page Not Found</h1>
            <p>We're sorry that you requested page could not be found. Please go back to the 
                <Link className={styles.link} to={"/"}> Home page </Link> 
                or you can contact us at <a className={styles.link} href="mailto:shopeeservice@shopee.com">shopeeservice@shopee.com</a>
            <img className={styles.image} src="https://i.ibb.co/kXB0hxJ/404.jpg" alt="404 warning" />
            </p>
        </div>
    )
}

export default Error