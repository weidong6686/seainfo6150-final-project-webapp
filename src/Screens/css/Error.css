
.container {
    margin: 2rem 8% 2rem 15%;
  }
  
  .image {
    height: 80%;
  }