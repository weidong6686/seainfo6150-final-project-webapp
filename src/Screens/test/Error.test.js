import React from "react";
import Error from "../Error.jsx";

describe("Error tests", () => {
  it("renders correctly", () => {
    const { container } = (<Error />);
    expect(container).toMatchSnapshot();
  });
});