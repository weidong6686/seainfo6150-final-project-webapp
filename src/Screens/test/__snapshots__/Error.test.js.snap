// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error tests renders correctly 1`] = `undefined`;

exports[`OrderConfirm tests renders correctly 1`] = `undefined`;
